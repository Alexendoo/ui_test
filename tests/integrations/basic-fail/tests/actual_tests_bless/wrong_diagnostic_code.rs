#![deny(dead_code, unreachable_code)]

fn foo() -> i32 {
    //~^ should_be_dead_code
    panic!();
    0 //~ unreachable_code
}

fn main() {}
