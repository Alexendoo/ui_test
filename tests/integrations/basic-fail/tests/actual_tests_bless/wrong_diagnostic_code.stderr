error: unreachable expression
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:6:5
  |
5 |     panic!();
  |     -------- any code following this expression is unreachable
6 |     0
  |     ^ unreachable expression
  |
note: the lint level is defined here
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:1:20
  |
1 | #![deny(dead_code, unreachable_code)]
  |                    ^^^^^^^^^^^^^^^^

error: function `foo` is never used
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:3:4
  |
3 | fn foo() -> i32 {
  |    ^^^
  |
note: the lint level is defined here
 --> tests/actual_tests_bless/wrong_diagnostic_code.rs:1:9
  |
1 | #![deny(dead_code, unreachable_code)]
  |         ^^^^^^^^^

error: aborting due to 2 previous errors

